osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -249,11 +262,13 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                     },
                     email=False
                 )
+                logger.exception()
```

Comment:
exception() will add stack trace and information about the exception by itself. You just need to pass a message.